README.md

```diff
@@ -432,7 +433,8 @@ _An Object containing the follow keys:_
 |__`query`__|_string_|Optional|
 |__`username`__|_string_|Optional|
 |__`featured`__|_boolean_|Optional|
-|__`category`__|_Array<number>_|Optional|
+|__`collections`__|_Array<number>_|Optional|
+|__`count`__|_string_|Optional|
```

Comment:
Similarly, thanks for adding the missing `collections` param!

Could I trouble you to re-add the `category` param? While it's deprecated, it _is_ still valid. At some point I imagine we'll launch a major version that drops support for it, but for now we should still keep it documented. Maybe adding an inline `deprecated` would help, beside the param name:

```
|__`category`__ *(deprecated)*|_Array<number>_|Optional|
```